specification/directories.adoc

```diff
@@ -0,0 +1,40 @@
+// Definitions of the directory structure to ensure that relative
+// links between ADOC files in sibling directories can be resolved.
+ifdef::env-github[]
+:url-specification: ../
+:url-specification-implicittiling: {url-specification}ImplicitTiling/
+:url-specification-implicittiling-availability: {url-specification-implicittiling}/AVAILABILITY.adoc
+:url-specification-metadata: {url-specification}Metadata/
+:url-specification-metadata-referenceimplementation: {url-specification-metadata}ReferenceImplementation/
+:url-specification-metadata-referenceimplementation-propertytable: {url-specification-metadata-referenceimplementation}PropertyTable/
+:url-specification-metadata-referenceimplementation-schema: {url-specification-metadata-referenceimplementation}Schema/
+:url-specification-metadata-semantics: {url-specification-metadata}Semantics/
```

Comment:
3D Metadata Semantic Reference should probably be an official section rather than an appendex

![image](https://user-images.githubusercontent.com/915398/170699316-ee07fdd6-867e-4ea4-8329-cd8f27b959f8.png)

Reply:
The reason why I moved it into the Appendix is that I thought that it should not be part of the _Specification of the format itself_. I saw it rather as a convention/vocabulary that is specific for the _application_ that consumes the data.

Of course, one could just move it into a section, and probably (?) tag that section as "non-normative", if this is desired...

Somehow, this is related to https://github.com/CesiumGS/3d-tiles/issues/643 ...